meson.build

```diff
@@ -1432,8 +1432,16 @@ vaapi_wayland = {
     'deps': dependency('libva-wayland', version: '>= 1.1.0', required: get_option('vaapi-wayland')),
 }
 vaapi_wayland += {'use': vaapi['use'] and egl_wayland['use'] and vaapi_wayland['deps'].found()}
-if vaapi_wayland['use']
-    features += vaapi_wayland['name']
+
+vaapi_wayland_memfd = {
+    'name': 'vaapi-wayland-memfd',
+    'deps': dependency('libva-wayland', version: '>= 1.1.0', required: get_option('vaapi-wayland')),
+}
+vaapi_wayland_memfd += {'use': vaapi_wayland['use'] and memfd_create}
```

Comment:
You don't need this dictionary object here. The dep was already checked above, so you can just reuse the vaapi_wayland['use']  bool.

Reply:
Thanks, not sure which dictionary object you're referring to. Are you saying I don't need the `deps` entry because I am using `vaapi_wayland` dependency ?

Reply:
Right, I mean that `vaapi_wayland_memfd` is not needed. You can just do `if vaapi_wayland['use'] and memfd`.